cpg-language-cxx/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/cxx/DeclaratorHandler.kt

```diff
@@ -168,9 +167,14 @@ class DeclaratorHandler(lang: CXXLanguageFrontend) :
 
         val func =
             when {
+                // Check if it's an operator. In this case, the name begins with operator
+                name.localName.startsWith("operator") -> {
```

Comment:
Some change is needed here: the comparison of operator with language trait operator names